server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -131,14 +127,15 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
             throw error;
         }
 
+        opMessages = [];
```

Comment:
(nit) move the default opMessages = [] declaration to where it's defined on line 75